cocli/README.md

```diff
@@ -0,0 +1,332 @@
+CBOR-encoded CoMIDs and/or CoSWIDs to be embedded.  For example:
+```
+$ cocli corim create --template c1.json --comid m1.cbor --coswid s1.cbor
+```
+On success, you should see something like the following printed to stdout:
+```
+>> created "c1.cbor" from "c1.json"
+```
+
+The CBOR-encoded CoRIM file is stored in the current working directory with a
+name derived from its template.  If you want, you can specify a different
+file name using the `--output` command line switch (abbrev. `-o`):
+```
+$ cocli corim create -t c1.json -m m1.cbor -s s1.cbor -o my.cbor
+>> created "my.cbor" from "c1.json"
+```
+
+CoMIDs and CoSWIDs can be either supplied as individual files, using the
+`--comid` (abbrev. `-m`) and `--coswid` (abbrev. `-s`) switches respectively, or
+as "per-folder" blocks using the `--comid-dir` (abbrev. `-M`) and `--coswid-dir`
+(abbrev. `-S`) switch.  For example:
+```
+$ cocli corim create --template c1.json --comid-dir comids.d/
+```
+
+Creation will fail if *any* of the inputs is non conformant:
+```
+$ cocli corim create -t c1.json -M comids.d/
+Error: error loading CoMID from comids.d/rubbish.cbor: EOF
+```
+
+### Sign
+
+Use the `corim sign` subcommand to cryptographically seal the unsigned CoRIM
+supplied via the `--file` switch (abbrev. `-f`).  The signature is produced
+using the key supplied via the `--key` switch (abbrev. `-k`), which is expected
+to be in [JWK](https://www.rfc-editor.org/rfc/rfc7517) format.  On success, the
+resulting COSE Sign1 payload is saved to file whose name can be controlled using
+the `--output` switch (abbrev. `-o`).  A CoRIM Meta<sup>[1](#templates-ex)</sup>
+template in JSON format must also be provided using the `--meta` switch (abbrev.
+`-m`).  For example, with the default output file:
+```
+$ cocli corim sign --file corim.cbor --key ec-p256.jwk --meta meta.json
+>> "corim.cbor" signed and saved to "signed-corim.cbor"
+```
+Or, the same but with a custom output file:
+```
+$ cocli corim sign --file corim.cbor \
+                 --key ec-p256.jwk \
+                 --meta meta.json \
+                 --output /var/spool/signed-corim.cbor
+>> "corim.cbor" signed and saved to "/var/spool/signed-corim.cbor"
+```
+
+### Verify
+
+Use the `corim verify` subcommand to cryptographically verify the signed CoRIM
+supplied via the `--file` switch (abbrev. `-f`).  The signature is checked
+using the key supplied via the `--key` switch (abbrev. `-k`), which is expected
+to be in [JWK](https://www.rfc-editor.org/rfc/rfc7517) format.  For example:
+```
+$ cocli corim verify --file signed-corim.cbor --key ec-p256.jwk
+>> "corim.cbor" verified
+```
+
+Verification can fail either because the cryptographic processing fails or
+because the signed payload or protected headers are themselves invalid.  For example:
+```
+$ cocli corim verify --file signed-corim-bad-signature.cbor --key ec-p256.jwk
+Error: error verifying signed-corim-bad-signature.cbor with key ec-p256.jwk: verification failed ecdsa.Verify
```

Comment:
I think, line 239 is the output, do you want it to be displayed like line 232, with >> 

Initially I struggled to understand whether line 232 is part of command or it is the output, then managed to see that.

Reply:
```suggestion
```
will return
```
Error: error verifying signed-corim-bad-signature.cbor with key ec-p256.jwk: verification failed ecdsa.Verify
```